ntbs-service/wwwroot/css/autocomplete.scss

```diff
@@ -2,4 +2,5 @@
 .autocomplete__option {
     color: black;
     text-decoration: none;
+    margin-bottom: 0px;

```

Comment:
This was related to the other autocomplete dropdown ticket, another small issue Nancy mentioned